converters/common.js

```diff
@@ -119,6 +119,31 @@ const TuyaDataTypes = {
     bitmap: 5, // [ 1,2,4 bytes ] as bits
 };
 
+const silvercrestModes = {
+    white: 0,
+    color: 1,
+    scene: 2,
```

Comment:
`effect` fits better then `scene` here

Reply:
I used the same name as the Lidl Home android App does. Still change it?

Reply:
In that case lets keep `scene`.